src/ProjectTemplates/test/AssemblyInfo.AssemblyFixtures.cs

```diff
@@ -1,4 +1,11 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
 using Microsoft.AspNetCore.E2ETesting;
 using ProjectTemplates.Tests.Helpers;
+using Xunit;
+
+[assembly: TestFramework("Microsoft.AspNetCore.E2ETesting.XunitTestFrameworkWithAssemblyFixture", "ProjectTemplates.Tests")]
 
 [assembly: AssemblyFixture(typeof(ProjectFactoryFixture))]
+[assembly: AssemblyFixture(typeof(SeleniumStandaloneServer))]
```

Comment:
We don't ask for the server anywhere but this way it gets setup and torn down when the tests start and end